indico/modules/events/editing/client/js/management/editable_type/EditableTypeDashboard.module.scss

```diff
@@ -5,11 +5,17 @@
 // modify it under the terms of the MIT License; see the
 // LICENSE file for more details.
 
+@import 'base/palette';
+@import 'base/typography';
+
 .toolbar-checkbox {
     align-self: center;
-}
+    margin-top: 0.4rem;
 
-.toolbar-label {
-    align-self: center;
-    padding: 0 1rem 0 0.5rem;
+    &:global(.ui.toggle.checkbox) input ~ label {
+        @extend %font-family-body;
+        color: $gray !important;
```

Comment:
I personally don't think that the color should be lighter here. I think the color provided by `action-box` css is more visible. Especially since we make the font even smaller in this case.

Reply:
Isn't it the same color? Without it, it's just black.

Reply:
![Bildschirmfoto 2020-04-09 um 17 48 36](https://user-images.githubusercontent.com/23189858/78914202-619d5d80-7a8a-11ea-90a7-792851a79775.png)
This is how it looks like for me when I remove the `color: $gray !important`. Alternatively I could try a darker gray than `$gray`?

Reply:
It wasn't black before. There was css being applied through `action-box` which was a darker version of grey... You can check the screenshot in the closed PR: https://github.com/indico/indico/pull/4380

Reply:
I moved the label into the Checkbox to make the label clickable as well, and that somehow made it black. But it seems like action-box did use $dark-gray, so I'll use that as well then! That should make it more visible.